recipe/meta.yaml

```diff
@@ -1,56 +1,78 @@
 {% set name = "python-lsp-server" %}
 {% set version = "1.5.0" %}
+{% set build_number = 1 %}
 
 package:
-  name: {{ name|lower }}
+  name: python-lsp-server-meta
```

Comment:
Is this really necessary? I mean, there's `qtconsole-base` also in the Qtconsole feedstock:

https://github.com/conda-forge/qtconsole-feedstock/blob/main/recipe/meta.yaml

but there's no need of an extra `meta` package.

Reply:
I've followed [jupyter-lsp-feedstock](https://github.com/conda-forge/jupyter-lsp-feedstock/blob/c3f3d8ab11b2668a8d8e9eb9b1a7cd3cef737e3a/recipe/meta.yaml#L15).
Apparently it otherwise create a name clash that would skip the sub-package tests.

The name here does not generate any package so we can put which ever name we like.